### ISSUE TITLE
AWS: hal deploy apply is not starting up apache correctly

### ISSUE THREAD

**Issue by ayk33** [spinnaker/spinnaker#2210]
Hello,

AWS: hal deploy apply is not starting up apache correctly

AWS

I am trying to install spinnaker via halyard in localdebian mode on an EC2 instance running Ubuntu 14.04 with an IAM role for all the permissions it needs. 

Running through the steps located on https://www.spinnaker.io/setup/install/halyard/ I eventually get to the `hal deploy apply` step. That seems to finish successfully (got exit code 0).

However when I run `netstat` there is nothing listening on port 9000. Scrolling through the `hal deploy apply` output I noticed there was an issue with apache:

```
Enabling site 000-default.
 * Starting web server apache2
 *
Setting up ssl-cert (1.0.33) ...
Processing triggers for ureadahead (0.100.0-16) ...
Processing triggers for ufw (0.34~rc-0ubuntu2) ...
Setting up spinnaker-deck (2.0.0-20171108151529) ...
Processing triggers for libc-bin (2.19-0ubuntu6.13) ...
null: unrecognized service
Enabling site spinnaker.
To activate the new configuration, you need to run:
  service apache2 reload
Site 000-default disabled.
To activate the new configuration, you need to run:
  service apache2 reload
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  spinnaker-front50
```
After running `sudo service apache2 reload` I was able to proceed. It looks like there might be some kind of race condition because the first occurrence of the line `null: unrecognized service` appears before apache is installed. 

Here are all the install steps to reproduce from a fresh EC2 instance:
```
sudo apt-get update
curl -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/stable/InstallHalyard.sh 
sudo bash InstallHalyard.sh -y. 
hal config storage s3 edit --region us-west-2
hal config storage edit --type s3
hal config provider aws account add test_account --account-id xxxxxxxxxxxx --assume-role arn:aws:iam::xxxxxxxxxxxx:role/halyard_test
hal config provider aws enable
hal config version edit --version 1.5.0
sudo hal deploy apply
```

**Comment by lwander** [comment]
https://github.com/spinnaker/halyard/pull/793

released in 0.39.0

**Comment by xacaxulu** [comment]
+1
